_data/transport_integrations.yml

```diff
@@ -0,0 +1,51 @@
+- type: Collector
```

Comment:
don't forget Kafka 0.8 and Scribe from the main repository

Reply:
Kafka 0.8 and Scribe are added.